apps/pr-deploy-site/.eslintrc.json

```diff
@@ -14,7 +14,7 @@
       // IE 11-compatible syntax as long as we still support IE 11.
       "files": ["pr-deploy-site.js"],
       "plugins": ["es"],
-      "extends": ["plugin:es/restrict-to-es2015"],
+      "extends": ["plugin:es/restrict-to-es2015", "plugin:@microsoft/eslint-plugin-sdl/required"],
```

Comment:
What was the reason for adding this plugin only to the PR deploy site?

Reply:
Because there are use cases of innerHTMl which is an SDL violation I had to suppress them.
EsLint fails if you have suppressions for plugins it doesn't know about.. 
I opted to create a scoped one to just this script.

Reply:
See: https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=239069&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55&t=cc94c182-d6f4-5dae-734d-3676c12cee2a

```
ERR! /mnt/vss/_work/1/s/apps/pr-deploy-site/pr-deploy-site.js
ERR!   78:5  error  Definition for rule '@microsoft/sdl/no-inner-html' was not found  @microsoft/sdl/no-inner-html
ERR!   86:5  error  Definition for rule '@microsoft/sdl/no-inner-html' was not found  @microsoft/sdl/no-inner-html
ERR!   98:5  error  Definition for rule '@microsoft/sdl/no-inner-html' was not found  @microsoft/sdl/no-inner-html
ERR! 
```